packages/web-components/src/theme/set-theme.ts

```diff
@@ -0,0 +1,12 @@
+import type { Theme } from '@fluentui/react-theme';
+import * as tokens from './design-tokens';
+
+/**
+ * Sets the theme tokens on defaultNode.
+ * @param theme Flat object of theme token values.
+ */
+export const setTheme = (theme: Theme) => {
```

Comment:
Only `defaultNode` (global) for now.
If agreed, we will add `setValueFor` version for scoped theming.